encoding/bytesutil/bytes_test.go

```diff
@@ -516,3 +517,9 @@ func TestSafeCopy2d32Bytes(t *testing.T) {
 	assert.Equal(t, false, &input == &output, "No copy was made")
 	assert.DeepEqual(t, input, output)
 }
+
+func TestNonZeroRoot(t *testing.T) {
+	input := make([]byte, fieldparams.RootLength)
+	output := bytesutil.NonZeroRoot(input)
+	assert.Equal(t, false, output)
+}
```

Comment:
Test the true case? This method could be implemented like this and still pass your test
```go
func NonZeroRoot(root []byte) bool {
  return false
}
```

Reply:
added cases for both false and true